cmd/ghalistener/metrics/metrics.go

```diff
@@ -144,75 +144,25 @@ var (
 		completedJobsTotalLabels,
 	)
 
-	jobStartupDurationSeconds = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
+	jobLastStartupDurationSeconds = prometheus.NewGaugeVec(
```

Comment:
worth adding a comment (in addition to commit message) why Gague is used, while ideally Histogram seems better data type - might avoid a lot of WTFs and wasting time basically reverting this change ;)